backend/LeetWars.Emailer/Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-focal AS base
```

Comment:
why did you change this?

Reply:
every out dockerfile has this, so I thought it will resolve sonarcloud security hotspot conflict